src/patternfly/patternfly-no-globals.scss

```diff
@@ -1,4 +1,5 @@
-$pf-v6-global--enable-reset: false;
-$pf-v6-global--enable-normalize: false;
-
-@import "./patternfly";
+@use './config/helpers' with (
+  $pf-v6-global--enable-reset: false,
+  $pf-v6-global--enable-normalize: false
+);
+@use './patternfly-base.scss';
```

Comment:
```suggestion
@use './patternfly.scss';
```

Reply:
https://sass-lang.com/dart-sass/#java-script-library the [sass package compiler ](https://sass-lang.com/dart-sass/#java-script-library) is slower than [using dart sass compiler](https://sass-lang.com/dart-sass/#dart-library). I'm happy to update the yarn script and test https://docs.google.com/document/d/1wd_ppsKREAibZlrhn5v3-BlvJgD71s-vBWYIQ31aok0/edit#heading=h.bsad7uswhrt.